### ISSUE TITLE
Statistical errors

### ISSUE THREAD

**Issue by ershockley** [ershockley/SPE#5]
We can now propagate systematic uncertainties fairly easily, but we need to work on the statistical uncertainties. This is fairly tricky due to correlated errors, so we will probably have to look up in books/literature how to do it. Steps are as follows:
 - LED and noise run have normal poisson errors for each bin
 - after subtraction LED-noise, can propagate in standard way because the two runs are independent
 - now tricky part, the acceptance. The total error of the integral is doable, but how to propagate something like "integral from x to inf" / "integral of -inf to inf"? These two values are correlated with each other so cannot do standard undergrad error propagation. I've never had to do something like this before... has anyone else?

**Comment by ershockley** [comment]
@awalsh272  can you summarize how the statistical errors are being propagated here and then close the issue? Also cite the PR you made.

**Comment by awalsh272** [comment]
[Acceptance Errors nonum [Autosaved].pdf](https://github.com/ershockley/SPE/files/1921541/Acceptance.Errors.nonum.Autosaved.pdf)


The attached pdf contains a description of the overall method. Commit 134a4cb8c9e4c13a9cc8b358002ea3834ab91672 contains completed error propagation, but later commits have it as well.
